conf/messages.en

```diff
@@ -3710,3 +3710,5 @@ assetDisposal.reportedOtherAssetsDisposalList.view.none.value = No
 removeAssetDisposal.title = Are you sure you want to remove details of the disposal of this asset?
 removeAssetDisposal.heading = Are you sure you want to remove details of the disposal of {0}?
 removeAssetDisposal.error.required = Select yes if you are sure you want to remove this asset disposal
+
+uploadMemberDetails.error.failed = The file upload failed. Please try again.
```

Comment:
It is a very minor thing, but if you are squashing commits anyway... could you please move this up in the same file, where the other uploadMemberDetails messages are?

Reply:
Done 👍